Sources/StreamChat/Query/Sorting/SortValue.swift

```diff
@@ -47,7 +47,7 @@ extension Array {
         } else if let lDate = lhs as? Date, let rDate = rhs as? Date {
             return isAscending ? lDate < rDate : lDate > rDate
         } else if let lBool = lhs as? Bool, let rBool = rhs as? Bool {
-            return isAscending ? !lBool && rBool : lBool && !rBool
+            return isAscending ? lBool && !rBool : !lBool && rBool
```

Comment:
Ascending should be from false to true 🤔

Reply:
Resolving, since it was discussed on planning 👍

Reply:
Can we add a comment here?

Reply:
@polqf ✅